src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -558,6 +558,10 @@ private function registerTemplatingConfiguration(array $config, $ide, ContainerB
                 'Symfony\\Bundle\\FrameworkBundle\\Templating\\Loader\\FilesystemLoader',
             ));
         }
+
+        if ($container->hasDefinition('assets.packages')) {
+            $container->getDefinition('templating.helper.assets')->replaceArgument(0, new Reference('assets.packages'));
```

Comment:
Though I didn't find circumstances where this ever will happen. If anyone can think of it, we could add a test covering it.